### ISSUE TITLE
tilemaker-server terminated by signal SIGKILL (Forced quit)

### ISSUE THREAD

**Issue by deevroman** [systemed/tilemaker#699]
```
tilemaker-server
fish: Job 1, 'tilemaker-server' terminated by signal SIGKILL (Forced quit)
```

Installed using this guide: https://github.com/systemed/tilemaker/blob/master/docs/INSTALL.md#macos

macOS: 14.4
Apple M1 Pro
RAM: 16GB

**Comment by seventyeight** [comment]
Same problem here. Build is running through without  problems. But executing the binary is causing the error above.

**Comment by systemed** [comment]
I haven't yet been able to reproduce this (macOS 14.1, Mac mini M2). Would welcome any reports that might indicate a possible cause.

**Comment by systemed** [comment]
Ah, interesting. It appears to be a LuaJIT thing. If I build with plain Lua it's fine. If I build with LuaJIT then it's killed.

Try uninstalling LuaJIT and installing plain Lua and see if that works for you.

**Comment by deevroman** [comment]
Yes, deleting it helped `brew uninstall --ignore-dependencies luajit` Is it possible to pass a build parameter that disables LuaJIT without deleting it?

**Comment by systemed** [comment]
I think the fix will be altering the LDFLAGS in the Makefile. I'll take a look at that.

**Comment by systemed** [comment]
#716 should fix this - do try it and let me know if it works for you.

**Comment by deevroman** [comment]
I updated the source code, installed luajit again and the problem is no longer there. Thanks!

**Comment by systemed** [comment]
That's great!

(For background: LuaJIT used to require a particular linking option on MacOS. With 2.1 it no longer does, so I've added a version check.)
